webapp/packages/plugin-administration/src/ConfigurationWizard/Welcome/WelcomePage.module.css

```diff
@@ -1,14 +1,8 @@
-/*
- * CloudBeaver - Cloud Database Manager
- * Copyright (C) 2020-2024 DBeaver Corp and others
- *
- * Licensed under the Apache License, Version 2.0.
- * you may not use this file except in compliance with the License.
- */
-
-.p {
+p {
```

Comment:
why has class now changed to a global style? this is CSS module we should not use global selectors like that

Reply:
fixed